test/purchases_flutter_test.dart

```diff
@@ -639,11 +639,10 @@ void main() {
         null,
         null,
       );
-      final purchasePackageResult =
-          await Purchases.purchaseSubscriptionOption(mockSubscriptionOption);
+      await Purchases.purchaseSubscriptionOption(mockSubscriptionOption);
 
       fail('an exception should have been thrown');
```

Comment:
should we be updating some unit tests to use the new ProductChangeInfo?

Reply:
Good call 😅 Will add!